mappings/net/minecraft/client/options/CloudRenderMode.mapping

```diff
@@ -0,0 +1 @@
+CLASS cqo net/minecraft/client/options/CloudRenderMode
```

Comment:
Is this used globally as an enum/constant set of sorts, or just as an option?

Reply:
It does get called in WorldRenderer to see if the current setting is OPTIONS_CLOUDS_FANCY, i assume to do its fancy clouds rendering.